src/main/java/frc/robot/subsystems/Chamber.java

```diff
@@ -109,14 +110,28 @@ public void stopChamber() {
     }
   }
 
+  public void calibrateChamber() {
+    setSpeed(m_calibrateSpeed.get());
+    int i = 0;
+    while (i < 1) {
+      if (!m_noteChamberedSensor.get()) i++;
+      Thread.yield();
+    }
+    // Continues to feed note back until not detected by sensor
+    while (!m_noteChamberedSensor.get()) Thread.yield();
+    stopChamber();
+  }
+
```

Comment:
This sort of busy-waiting doesn't work, because we need to give control back to the main scheduler loop ASAP.